### ISSUE TITLE
Rendering artifacts on some shapes

### ISSUE THREAD

**Issue by lowercase93** [dmester/jdenticon-net#4]
Hi,
Great library! I realy like it and use it all over my app.
But I notice sometimes rendering artifacts that appear on some kinds of shapes. Here are samples (I hope github won't be compressing the image. click it to view full size):
![artifacts](https://user-images.githubusercontent.com/10177470/49340078-96520700-f643-11e8-91be-a9f4e32252b8.png)

Here is the code I use to generate images:
```
const int size = 128;
const int value = 0x5D503474; // the value below the samples in the pic above.
Identicon icon = Identicon.FromValue(value, size);
icon.Style.BackColor = Color.Transparent; // the darkgray background in samples is just a WPF window background.
icon.Style.Padding = 0f;
```

Then i just convert it to bitmap image to be able to display on window:
``` 
using (var stream = icon.SaveAsPng())
{
    var bitmapImage = new BitmapImage();
    bitmapImage.BeginInit();
    bitmapImage.CacheOption = BitmapCacheOption.OnLoad;
    bitmapImage.StreamSource = stream;
    bitmapImage.EndInit();
    bitmapImage.Freeze();
    return bitmapImage;
} 
```

**Comment by dmester** [comment]
Thank you for the detailed bug report. I have investigated the issue and reproduced the artifacts. They seem to be triggered by having no padding in the icons. In this case the color of the last pixel per row was leaking into the following row producing the artifacts you have seen.

I have created a fix for the issue and published a new release (2.2.1).

**Comment by lowercase93** [comment]
That was fast! Thank you a lot!
